tests/python/contrib/test_cmsisnn/test_conv2d.py

```diff
@@ -304,6 +304,94 @@ def test_conv2d_asymmetric_padding_int8(
     )
 
 
+# This test expects assertion as the output should mismatch between TVM and CMSIS-NN
```

Comment:
Could this assert that it matches the expected output? Similar to how the NPU tests check that they are consistent with TFLite?

Reply:
Yes, we could compare against TFLite output. So far all the CMSIS-NN tests compare against TVM output.